pkgs/os-specific/linux/adcli/default.nix

```
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, fetchpatch
+, openldap
+, libkrb5
+, libxslt
+, autoreconfHook
+, pkg-config
+, cyrus_sasl
+, util-linux
+}:
+
+stdenv.mkDerivation rec {
+  pname = "adcli";
+  version = "0.9.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "realmd";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-Zzt4qgLiJNuSrbtDWuxJEfGL7sWSbqN301q3qXZpn9c=";
+  };
+
+  # https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=01db544de893262e857685e11b33c2a74210181f
```

Unneeded line - it essentially repeats below at the expression.
Maybe the bugs entry onm gentoo is better

```suggestion
   # https://bugs.gentoo.org/820224
```

---

It stays the same?